lib/TPP/ConvertCheckToFunc.cpp

```diff
@@ -0,0 +1,172 @@
+//===- ConvertCheckToFunc.cpp -------------------------------------*- C++-*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "TPP/Dialect/Check/CheckOps.h"
+#include "TPP/Passes.h"
+#include "TPP/Transforms.h"
+#include "mlir/Dialect/Arith/IR/Arith.h"
+#include "mlir/Dialect/Func/IR/FuncOps.h"
+#include "mlir/Dialect/LLVMIR/LLVMDialect.h"
```

Comment:
not needed

Reply:
I can only remove Arith header from this list. There are dependences on other headers that I cant get rid of.